java/src/jmri/jmrix/AbstractSerialPortController.java

```diff
@@ -262,6 +283,21 @@ final protected void configureLeads(com.fazecast.jSerialComm.SerialPort serialPo
 
     }
 
+    // Possible parity options
+    static final protected int PARITY_NONE = com.fazecast.jSerialComm.SerialPort.NO_PARITY;
+    static final protected int PARITY_EVEN = com.fazecast.jSerialComm.SerialPort.EVEN_PARITY;
+    static final protected int PARITY_ODD  = com.fazecast.jSerialComm.SerialPort.ODD_PARITY;
```

Comment:
@bobjacobsen 
Would you mind if I change these to an enum after this PR is merged? That would let the compiler check the value during compilation.

See the enum `TurnoutState`:
https://github.com/JMRI/JMRI/blob/master/java/src/jmri/jmrit/logixng/actions/ActionTurnout.java#L103

Reply:
Would be happy to have you change that!  

I used an enum for the FlowControl values in this class.  It's a nice solution.